package.json

```diff
@@ -33,26 +33,26 @@
   "author": "Juan Cruz Viotti <juan@resin.io>",
   "license": "Apache-2.0",
   "devDependencies": {
-    "electron-mocha": "^3.3.0",
-    "eslint": "^3.14.0",
-    "jsdoc-to-markdown": "^2.0.1",
-    "mocha": "^3.2.0",
+    "electron-mocha": "^3.5.0",
+    "eslint": "^3.19.0",
+    "jsdoc-to-markdown": "^5.0.3",
+    "mocha": "^3.5.3",
     "mochainon": "^1.0.0"
   },
   "dependencies": {
-    "bluebird": "^3.4.7",
+    "bluebird": "^3.7.2",
     "bluebird-retry": "^0.10.1",
-    "bmapflash": "^1.2.1",
+    "bmapflash": "^1.2.2",
```

Comment:
@zvin it's used here

Reply:
Rummaging around, I found https://github.com/balena-io-modules/etcher-image-write/pull/105#issuecomment-453186702 :wink: